app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<html class="Reg_Html">
+<body class="Reg_Body">
+<div class="container">
+  <div class="row">
+    <div class="col">
+    </div>
+  </div>
+</div>
+<%= form_with url: home_pages_index_url, method: :push do |form| %>
```

Comment:
Das ist schon besser, aber immer noch nicht richtig. In Formular wird bei Rails immer ein Model benutzt.